src/main/fc/init.c

```diff
@@ -984,6 +984,11 @@ void init(void)
     motorEnable();
 #endif
 
+    // MOTOR_STOP is not used when AIRMODE is enabled, note AIRMODE can be on a switch
```

Comment:
What if airmode is on a switch? init() only runs on startup, so motor stop would probably not get deactivated in case you activate airmode only after turning on the FC.